avro2py/avro_types.py

```diff
@@ -0,0 +1,200 @@
+"""Representation of avro schema types."""
+from contextlib import suppress
+from enum import Enum as _Enum, auto, unique
+from typing import NamedTuple, List, Union, Optional, Any, NewType, Dict
+
+
+def case_insensitive(cls):
+    """Add case-insensitive .get method to Enum."""
+    @classmethod
+    def get(cls_, value):
+        return cls_[value.upper().replace('-', '_')]
+    cls.get = get
```

Comment:
fierce duck punch. 
maybe in the docstring, explain why this is necessary? or it's just for convenience?